tests/test_explain_model.py

```diff
@@ -906,15 +906,15 @@ def iris_per_class_expected_features(self):
                 ['petal length', 'petal width', 'sepal length', 'sepal width']]
 
     @property
-    def boston_local_features_first_five_rf(self):
+    def housing_local_features_first_five_rf(self):
         return [['LSTAT', 'CRIM', 'B', 'AGE', 'INDUS', 'RAD', 'CHAS', 'ZN', 'TAX', 'DIS', 'PTRATIO', 'NOX', 'RM'],
```

Comment:
I think all of these checks will need to be updated to the new features

Reply:
Done.